.github/workflows/publish.yml

```diff
@@ -40,22 +40,22 @@ jobs:
           - pluto_corrections
     steps:
       - uses: actions/checkout@v2
-      - uses: NYCPlanning/action-library-archive@v1.1
+      - uses: NYCPlanning/action-library-archive@v1.2-alpha
         if: matrix.dataset != 'pluto_corrections'
         with:
           name: ${{ matrix.dataset }}
           s3: true
-          latest: true
+          latest: false
```

Comment:
I guess I haven't thought about the DO side. I think it is definitely smart to not touch the latest for any of the datasets.

Reply:
Yea that was my thinking. Should be easy to fix by running publishing from the main branch if we do mess something up though